packages/menu/src/MenuList/MenuList.js

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import { themeGet } from 'styled-system'
+import { Flex, applySizes } from 'pcln-design-system'
+
+const sizes = {
+  singleColumn: css`
+    & > * {
+      width: 100%;
+    }
+  `,
+  twoColumns: css`
+    & > * {
+      width: calc(50% - ${themeGet('space.1')}px);
+    }
+  `,
+}
+
+const MenuContainer = styled(Flex)`
+  font-family: 'Montserrat', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  overflow-y: scroll;
+  height: ${(props) => props.height || 300}px;
```

Comment:
maybe rename this `heightPx`? this collides with the `styled-system` height that accepts percents and em/rem/viewport units

Reply:
I just removed this line, since it's a Flex component it accepts `height` as a prop and handles everything correctly